Network/DNS/IO.hs

```diff
@@ -70,7 +82,11 @@ receiveVC sock = do
     let len = case map ord $ LBS.unpack lenbytes of
                 [hi, lo] -> 256 * hi + lo
                 _        -> 0
+#if MIN_VERSION_conduit(1,3,0)
+    fst <$> (src $$+- CB.isolate len .| sink)
```

Comment:
The `.|` operator is available in conduit 1.2 as well (starting with 1.2.8), if you'd like to bypass the CPP here. Same applies to the implementation of `receive`.